dockerfile-image-update/src/test/java/com/salesforce/dockerfileimageupdate/model/GitForkBranchTest.java

```diff
@@ -61,11 +61,29 @@ public Object[][] sameBranchOrImageNamePrefix() {
 
     @Test(dataProvider = "sameBranchOrImageNamePrefix")
     public void testIsSameBranchOrHasImageNamePrefix(String imageName, String imageTag, String specifiedBranch, String inputBranch, boolean expectedResult) {
-        assertEquals(new GitForkBranch(imageName, imageTag, specifiedBranch).isSameBranchOrHasImageNamePrefix(inputBranch), expectedResult);
+        assertEquals(new GitForkBranch(imageName, imageTag, specifiedBranch, "Dockerfile,docker-compose").isSameBranchOrHasImageNamePrefix(inputBranch), expectedResult);
     }
 
     @Test(expectedExceptions = IllegalArgumentException.class)
     public void testIsSameBranchOrHasImageNamePrefix() {
-        new GitForkBranch("", "1", "");
+        new GitForkBranch("", "1", "", null);
+    }
+
+    @DataProvider
+    public Object[][] imageNameSpecifiedBranchAndFilenameSearched() {
+        return new Object[][]{
+                /*{"docker.io/some/container",     "",     "blah", "dockerfile", "blah"},
```

Comment:
Do you need these commented?

Reply:
That was added for some debugging, uncommented now. Thanks @afalko